css/src/atomics/border.scss

```diff
@@ -80,14 +80,26 @@
 
 .border-radius {
 	border-radius: $border-radius !important;
+
+	> [class^='background-color-'] {
+		border-radius: $border-radius-sm !important;
+	}
 }
 
 .border-radius-lg {
 	border-radius: $border-radius-lg !important;
+
+	> [class^='background-color-'] {
```

Comment:
border-radius shouldn't interact with background-color. want to avoid tangential effects with atomics